site-src/enhancements.md

```diff
@@ -0,0 +1,73 @@
+# Network Policy Enhancement Proposal (NPEP)
+
+Network Policy Enhancement Proposals (NPEP) serve a similar purpose to the [KEP][kep]
+process for the main Kubernetes project:
+
+1. Ensure that changes to the API follow a known process and discussion
+   in the OSS community.
+2. Make changes and proposals discoverable (current and future).
+3. Document design ideas, tradeoffs, decisions that were made for
+   historical reference.
+
+## Process
+
+### 1. Discuss with the community
+Before creating a NPEP, share your high level idea with the community. Check
+[Community, discussion, contribution, and support](/#community-discussion-contribution-and-support)
+for the suitable communication option.
+
+### 2. Create an Issue
+[Create a NPEP issue](https://github.com/kubernetes-sigs/network-policy-api/issues/new?assignees=&labels=kind%2Ffeature&template=enhancement.md) 
+in the repo describing your change.
+At this point, you should copy the outcome of any other conversations or documents
+into this Issue.
+
+### 3. Create a first PR for your NPEP
+NPEP process is supposed to be iterative, adding more details with every iteration.
+To start this process, create a PR adding `npep-<issue number>.md` file in [npep folder](https://github.com/kubernetes-sigs/network-policy-api/tree/master/npep)
+using [template NPEP](https://github.com/kubernetes-sigs/network-policy-api/blob/master/npep/npep-95.md). Further we describe a number of 
+suggested iterations for an NPEP to make sure every version has a specific goal and limited scope.
+
+### 4. Agree on the Goals
```

Comment:
So each of these steps will in theory be in a single PR, should we have some sort of PR naming convention i.e `GoalDevelopment-NPEP93` or something like that?

Reply:
That is what Statuses are used for in GEP, I have added 3 statuses for every stage, and Status field to NPEP to signal on which stage every NPEP is